app/graphql/mutations/create_dynamic_field.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class Mutations::CreateDynamicField < Mutations::BaseMutation
+  argument :string_key, ID, required: true
+  argument :display_label, String, required: true
+  argument :sort_order, Integer, required: true
+  argument :is_facetable, Boolean, required: true
```

Comment:
I don't think the `is_*` fields need to be required. The database will assign default values if they aren't provided.